modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -83,21 +113,52 @@ spec:
           name: cloud-credentials
         default: true
         objectStorage:
-          bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          bucket: my-bucket
+          prefix: velero
         provider: aws
   configuration:
     restic:
       enable: <true_or_false>
     velero:
+       itemOperationSyncFrequency: "10s"
       defaultPlugins:
         - openshift
         - aws
         - csi
+        - vsm <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14" <4>
+      volumeOptions: <5>
+        sourceVolumeOptions:
+          accessMode: ReadOnlyMany
+          cacheAccessMode: ReadWriteOnce
+          cacheCapacity: 2Gi
+          moverSecurityContext: true
```

Comment:
@CarmiWisemon I think we need to remove this field from documentation as it's removed recently.
https://issues.redhat.com/browse/OADP-1941